### PR TITLE
Closes #2576. Update link to security info on z.cash website.

### LINKED ISSUE #2576: README and CLI should both link to https://z.cash/security/
As [noted](https://chat.zcashcommunity.com/channel/zcash-dev?msg=HLrzXGv5jE2sq4B2W) by @zookozcash:

[README.md](https://github.com/zcash/zcash/blob/2a42478bff3e1298f73e0411eb6d69d83b89f6a0/README.md) links to [doc/security-warnings.md](https://github.com/zcash/zcash/blob/2a42478bff3e1298f73e0411eb6d69d83b89f6a0/doc/security-warnings.md), but the CLI links to <https://z.cash/security/index.html>.

* Ideally, the README should link to the website too so there is a single starting point for security warnings.
* The links should be shortened by dropping "index.html": <https://z.cash/security/>.

### PR DESCRIPTION
**Pull request by bitcartel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Security Warnings
 -----------------
 
 See important security warnings in
-[doc/security-warnings.md](doc/security-warnings.md).
+[Security Information](https://z.cash/support/security/).
```
Comment (str4d) [inline README.md]: `See important security warnings on the [Security Information page]()`.

Comment (bitcartel) [reply README.md]: Done

**Comment by bitcartel** [comment]
@daira Quick review?  Thanks.

**Comment by ebfull** [comment]
Can we avoid accidental redundancy/inconsistency between these two pages?

**Comment by arcalinea** [comment]
utACK

**Comment by str4d** [comment]
@ebfull not trivially, as we can't easily substitute anything into the README. If you want to investigate that, please open a new issue to track it.

@zkbot r+

**Pull request merged**